ansible/setup.yml

```diff
@@ -55,13 +55,3 @@
       with_items:
         - git
         - python-pip
-
-    # Temporary fix for https://github.com/aeternity/epoch/issues/270
```

Comment:
I understand this can be removed thanks to https://github.com/aeternity/epoch/blob/1be1208c561cfdae856e40ce6bf86ff42b4549ec/deployment/ansible/templates/epoch.yaml.j2#L11

Do you plan to manually delete those old lines from `/etc/hosts`?

Reply:
Yes, using the peer address is the right way.

All hosts files are already clean.